<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Operation</class>
 <widget class="QMainWindow" name="Operation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="Btn1_1">
    <property name="geometry">
     <rect>
      <x>211</x>
      <y>131</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="Btn4_1">
    <property name="geometry">
     <rect>
      <x>211</x>
      <y>301</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="Btn2_1">
    <property name="geometry">
     <rect>
      <x>211</x>
      <y>191</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="Btn3_1">
    <property name="geometry">
     <rect>
      <x>211</x>
      <y>251</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="Btn5_1">
    <property name="geometry">
     <rect>
      <x>211</x>
      <y>361</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="Btn6_1">
    <property name="geometry">
     <rect>
      <x>211</x>
      <y>411</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="led1">
    <property name="geometry">
     <rect>
      <x>141</x>
      <y>131</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="led2">
    <property name="geometry">
     <rect>
      <x>141</x>
      <y>192</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="led3">
    <property name="geometry">
     <rect>
      <x>141</x>
      <y>252</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="led4">
    <property name="geometry">
     <rect>
      <x>141</x>
      <y>303</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="led6">
    <property name="geometry">
     <rect>
      <x>141</x>
      <y>414</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QLabel" name="led5">
    <property name="geometry">
     <rect>
      <x>141</x>
      <y>363</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Btn5_2">
    <property name="geometry">
     <rect>
      <x>301</x>
      <y>361</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="Btn6_2">
    <property name="geometry">
     <rect>
      <x>301</x>
      <y>411</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="Btn1_2">
    <property name="geometry">
     <rect>
      <x>301</x>
      <y>131</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="Btn2_2">
    <property name="geometry">
     <rect>
      <x>301</x>
      <y>191</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="Btn3_2">
    <property name="geometry">
     <rect>
      <x>301</x>
      <y>251</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="Btn4_2">
    <property name="geometry">
     <rect>
      <x>301</x>
      <y>301</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="led1_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>130</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QLabel" name="led3_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>251</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QLabel" name="led2_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>191</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QLabel" name="led5_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>362</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>11</string>
    </property>
   </widget>
   <widget class="QLabel" name="led4_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>302</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="led6_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>413</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>12</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>130</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>气缸1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>191</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>气缸2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>251</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>气缸3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>302</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>伺服手动使能</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>501</x>
      <y>413</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>伺服手动</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>362</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>伺服自动</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Btn7_1">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>130</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="Btn8_1">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>190</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="Btn10_1">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>300</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="Btn9_1">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>250</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="Btn11_1">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>360</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="Btn12_1">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>410</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>131</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Epson开关</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>192</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>入库系统开关</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>252</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Epson吸盘</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>363</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>夹爪</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>303</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>传送带</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>414</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>剔除</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Btn8_2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>190</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="Btn7_2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>130</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="Btn9_2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>250</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="Btn10_2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>300</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="Btn11_2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>360</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="Btn12_2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>410</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="Btn14_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>520</y>
      <width>49</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>463</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>激光雕刻</string>
    </property>
   </widget>
   <widget class="QLabel" name="led7">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>463</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>13</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Btn14_1">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>520</y>
      <width>49</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Epson状态</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Btn13_1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>460</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="Btn13_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>460</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="led7_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>520</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>14</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Btn14_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>520</y>
      <width>49</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="Btn14_4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>520</y>
      <width>49</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="Btn14_5">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>520</y>
      <width>49</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="topiclabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Epson控制界面&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>892</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
